src/components/Dashboard/MainInput/MainInput.jsx

```diff
@@ -54,27 +36,22 @@ export default class MainInput extends React.Component {
     });
   };
 
-  setInputValue = (value) => {
+  setInputValue = (value = '') => {
```

Comment:
you can skip brackets if function takes only one param

Reply:
If I may, I would like to disagree here. According to the style guide we are using for this project (AirBnB rules), parameter brackets may be skipped only if a function does not have a curly braces body.

Reply:
good point, no problem from my side